src/setup/execute.c

```diff
@@ -68,9 +68,15 @@ static char *TempFile(const char *s)
         tempdir = ".";
     }
 #else
-    // In Unix, just use /tmp.
+    
+    // Check the TEMP environment variable to find the location.
```

Comment:
That'd be `TMPDIR`. Also, please drop the extra blank line above (same in `src/m_misc.c`).